src/plugins/controls/public/control_types/range_slider/range_slider.component.tsx

```diff
@@ -6,68 +6,12 @@
  * Side Public License, v 1.
  */
 
-import React, { FC, useCallback } from 'react';
-import { BehaviorSubject } from 'rxjs';
+import React from 'react';
 
-import { DataViewField } from '@kbn/data-views-plugin/public';
-import { useReduxEmbeddableContext } from '@kbn/presentation-util-plugin/public';
-import { useStateObservable } from '../../hooks/use_state_observable';
 import { RangeSliderPopover } from './range_slider_popover';
-import { rangeSliderReducers } from './range_slider_reducers';
-import { RangeSliderEmbeddableInput, RangeValue } from './types';
 
 import './range_slider.scss';
 
-interface Props {
-  componentStateSubject: BehaviorSubject<RangeSliderComponentState>;
-  ignoreValidation: boolean;
-}
-// Availableoptions and loading state is controled by the embeddable, but is not considered embeddable input.
-export interface RangeSliderComponentState {
-  field?: DataViewField;
-  fieldFormatter: (value: string) => string;
-  min: string;
-  max: string;
-  loading: boolean;
-  isInvalid?: boolean;
-}
-
-export const RangeSliderComponent: FC<Props> = ({ componentStateSubject, ignoreValidation }) => {
-  // Redux embeddable Context to get state from Embeddable input
-  const {
-    useEmbeddableDispatch,
-    useEmbeddableSelector,
-    actions: { selectRange },
-  } = useReduxEmbeddableContext<RangeSliderEmbeddableInput, typeof rangeSliderReducers>();
-  const dispatch = useEmbeddableDispatch();
-
-  // useStateObservable to get component state from Embeddable
-  const { loading, min, max, fieldFormatter, isInvalid } =
-    useStateObservable<RangeSliderComponentState>(
-      componentStateSubject,
-      componentStateSubject.getValue()
-    );
-
-  const { value, id, title } = useEmbeddableSelector((state) => state);
-
-  const onChangeComplete = useCallback(
-    (range: RangeValue) => {
-      dispatch(selectRange(range));
-    },
-    [selectRange, dispatch]
-  );
-
-  return (
-    <RangeSliderPopover
-      id={id}
-      isLoading={loading}
-      min={min}
-      max={max}
-      title={title}
-      value={value ?? ['', '']}
-      onChange={onChangeComplete}
-      fieldFormatter={fieldFormatter}
-      isInvalid={!ignoreValidation && isInvalid}
-    />
-  );
+export const RangeSliderComponent = () => {
+  return <RangeSliderPopover />;
```

Comment:
Hmm... Is it worth even keeping `RangeSliderComponent` at this point now that a bunch of logic has been removed? Could we not just use `RangeSliderPopover` component directly for the embeddable render?

Reply:
Created this [issue](https://github.com/elastic/kibana/issues/137888)